benchmarks/TerraFX.Interop.Mimalloc.AllocatorsBenchmarks.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Should multi-target net5.0 until net6.0 ships and I move to that exclusively.

Reply:
But then we aren't getting results for `NativeMemory`

Reply:
Then its probably fine to keep it targeting net6.0.

IIRC, BDN doesn't work well in that scenario.